menpo/shape/mesh/coloured.py

```diff
@@ -162,38 +171,169 @@ def from_mask(self, mask):
             ctm.colours = ctm.colours[isolated_mask, :]
             return ctm
 
-    def _view_3d(self, figure_id=None, new_figure=False, coloured=True,
-                 **kwargs):
+    def with_clipped_texture(self, range=(0., 1.)):
+        """
+        Method that returns a copy of the object with the coloured values
+        clipped in range ``(0, 1)``.
+
+        Parameters
+        ----------
+        range : ``(float, float)``, optional
+            The clipping range.
+
+        Returns
+        -------
+        self : :map:`ColouredTriMesh`
+            A copy of self with its texture clipped.
+        """
+        instance = self.copy()
+        instance.colours = np.clip(self.colours, *range)
+        return instance
+
+    def with_rescaled_texture(self, minimum, maximum, per_channel=True):
+        r"""A copy of this mesh with colours linearly rescaled to fit a range.
+
+        Parameters
+        ----------
+        minimum: `float`
+            The minimal value of the rescaled colours
+        maximum: `float`
+            The maximal value of the rescaled colours
+        per_channel: `boolean`, optional
+            If ``True``, each channel will be rescaled independently. If
+            ``False``, the scaling will be over all channels.
+
+        Returns
+        -------
+        coloured_mesh : ``type(self)``
+            A copy of this mesh with colours linearly rescaled to fit in the
+            range provided.
+        """
+        instance = self.copy()
+        colours = instance.colours
+        if per_channel:
+            min_, max_ = colours.min(axis=0), colours.max(axis=0)
+        else:
+            min_, max_ = colours.min(), colours.max()
+        sf = ((maximum - minimum) * 1.0) / (max_ - min_)
```

Comment:
Yeah that's redundant! 

Btw should we rename this method to `rescale_texture()` in order to be in correspondence with `rescale_pixels()` of `Image`?